MELA/python/mela_binding.cpp

```diff
@@ -215,6 +217,29 @@ SimpleParticleCollection_t collection_initializer(py::list listOfParticles){
     return collection;
 }
 
+void setInputEvent(Mela& mela, SimpleParticleCollection_t* daughters, SimpleParticleCollection_t* associated, SimpleParticleCollection_t* mothers, bool isgen){
+    if((daughters) && (daughters->size() == 0)){
+        daughters = 0;
+    }
+    if ((associated) && (associated->size() == 0)){
+        associated = 0;
+    }
+    if ((mothers) && (mothers->size() == 0)){
+        mothers = 0;
+    }
+    mela.setInputEvent(daughters, associated, mothers, isgen);
+}
+
+void PrintCurrentCandidateSummary(Mela& mela){
```

Comment:
Ok, but we should still have a PrintCandidateSummary version which receives a MELACandidate. MELACandidates do not need to be created within Mela (and for complex applications, they are not).

Reply:
Give me some time for this - in order to do this I have to fill out the implementation for `MELACandidate`, `MELAParticle`, and `MELAThreeBodyDecayCandidate.` I will make a commit after testing these implementations.